perly.y

```diff
@@ -1154,8 +1157,12 @@ termbinop:	term[lhs] PLUGHIGHOP[op] term[rhs]
 			{ $$ = newRANGE($DOTDOT, scalar($lhs), scalar($rhs)); }
 	|	term[lhs] ANDAND term[rhs]                       /* $x && $y */
 			{ $$ = newLOGOP(OP_AND, 0, $lhs, $rhs); }
+	|	term[lhs] PLUGLOGICALANDOP[op] term[rhs]
+			{ $$ = build_infix_plugin($lhs, $rhs, $op); }
 	|	term[lhs] OROR term[rhs]                         /* $x || $y */
 			{ $$ = newLOGOP(OP_OR, 0, $lhs, $rhs); }
+	|	term[lhs] PLUGLOGICALOROP[op] term[rhs]
```

Comment:
Does this handle DORDOR too?

Reply:
These are the custom infix plugin ops. There isn't a `PLUGDOR...` because it would just be at the same precedence level as regular OR ops anyway.